lib/game.rb

```diff
@@ -0,0 +1,18 @@
+class Game
+  def initialize(input:, output:)
+    @input = input
+    @output = output
+  end
+
+  def play
+    @output.puts <<~GRID
```

Comment:
I usually prefer attr_readers to for instance variable vs direct use. The reason why is because it allows us to not worry about whether it is data/state or a method. That gives us more design flexibility when we need to change it to a method, but also just reduces the cognitive load. 

That is a pretty minor thing though, and others have reasons for using them.

Reply:
Oh, I missed that! I probably would have realized it wasn't and attr the second time I used it, but it totally slipped by me. Updated in c693617890fde8abb79ba279d14851feed2c0b78